integration-tests/test_unregister.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_unregister(insights_client):
+    """Test insights-client --unregister after registering it
+
+    If the unregistration is successful, `unregister` method returns True
+    """
+    insights_client.register()
+    assert insights_client.is_registered
+
+    registration_status = insights_client.run("--unregister")
+    assert (
+        "Successfully unregistered from the Red Hat Insights Service"
+        in registration_status.stdout
+    )
```

Comment:
after this, you can also assert that is `not is_registered`

Reply:
ok I wil add this.